exo-go/src/app_deployer/index.go

```diff
@@ -0,0 +1,35 @@
+package appDeployer
+
+import (
+	"path/filepath"
+
+	"github.com/Originate/exosphere/exo-go/src/logger"
+	"github.com/Originate/exosphere/exo-go/src/terraform_file_helpers"
+	"github.com/Originate/exosphere/exo-go/src/types"
+)
+
+// AppDeployer contains information needed to deploy the application
+type AppDeployer struct {
+	AppConfig      types.AppConfig
+	ServiceConfigs map[string]types.ServiceConfig
+	Logger         *logger.Logger
+	AppDir         string
+	HomeDir        string
+	TerraformDir   string
+}
+
+// NewAppDeployer is the constructor for the appDeployer class
+func NewAppDeployer(appConfig types.AppConfig, serviceConfigs map[string]types.ServiceConfig, appDir, homeDir string) *AppDeployer {
+	return &AppDeployer{
+		AppConfig:      appConfig,
+		ServiceConfigs: serviceConfigs,
+		AppDir:         appDir,
+		HomeDir:        homeDir,
+		TerraformDir:   filepath.Join(appDir, "terraform"),
```

Comment:
Since this is based on appDir, how about having it be a computed property `getTerraformDir`? (can make it public if needed)